apps/src/aichat/types.ts

```diff
@@ -26,6 +26,7 @@ export enum Role {
   USER = 'user',
   SYSTEM = 'system',
   MODEL_UPDATE = 'update',
+  ERROR = 'error',
```

Comment:
"error" as a role feels weird, but I can't think of a better option. [Ties into your comment here](https://github.com/code-dot-org/code-dot-org/pull/58668#pullrequestreview-2071960253) about types of these messages, which I agree we could improve going forward.

Reply:
We could temporarily call it `ERROR_NOTIFICATION`.

Reply:
Yep I didn't feel great tacking on another role here but this name sounds good to me. Would definitely like to revisit these types and refactor a bit as we've discussed